tracing-futures/Cargo.toml

```diff
@@ -18,16 +18,15 @@ keywords = ["logging", "profiling", "tracing", "futures", "async"]
 license = "MIT"
 
 [features]
-default = ["futures-01", "tokio"]
-futures-01 = ["futures"]
-futures-03 = ["std-future", "futures-task"]
+default = ["futures-03", "tokio"]
```

Comment:
I'm not sure how useful the `futures` crate integrations are in practice...perhaps we should just enable the `std-future` integration by default? I could be convinced otherwise.